### ISSUE TITLE
Is AWB emulation support covers P4Gs voice over AWBs?

### ISSUE THREAD

**Issue by esqwrf** [Sewer56/FileEmulationFramework#20]
and is there any pitfalls?

**Comment by Sewer56** [comment]
I'm pretty sure it's been done before.

I think someone once requested some help because they ran into a caveat in one of the Persona games, but it turned out to be user error.

As for pitfalls, I guess a millisecond or two more delay playing if on a hard drive, else no difference if on SSD.

**Comment by esqwrf** [comment]
Then I would also like to ask for help.
![image](https://github.com/Sewer56/FileEmulationFramework/assets/121329480/7ad3a04a-2ebd-4aa7-9fa8-cd08e66fb136)
Here's my file structure. Doesn't work, I also tried to switch the VO language in the game. (is there any support for lang slots to avoid undesirable overrides if playing with JPN? All Eng AWBs stored in the EN subfolder in original CPK).

**Comment by esqwrf** [comment]
Yes indeed, the lack of ACB is to blame, somehow missed this in the log earlier, my previous post was a guess on this part.
